demo/tiles/squares-page.html

```diff
@@ -58,41 +58,40 @@
     properties: {
 
       sharedElements: {
-        value: function() {
-          return {
-            'hero': this.$.header
-          }
-        }
+        type: Object
       },
 
       animationConfig: {
-        value: function() {
-          var squares = Polymer.dom(this.root).querySelectorAll('.square');
-          var squaresArray = Array.prototype.slice.call(squares);
-          return {
-            'entry': [{
-              name: 'hero-animation',
-              id: 'hero',
-              toPage: this
-            }, {
-              name: 'cascaded-animation',
-              animation: 'transform-animation',
-              transformFrom: 'translateY(100%)',
-              nodes: squaresArray
-            }],
-
-            'exit': [{
-              name: 'slide-up-animation',
-              node: this.$.header
-            }, {
-              name: 'cascaded-animation',
-              animation: 'transform-animation',
-              transformTo: 'translateY(60vh)',
-              nodes: squaresArray
-            }]
-          };
-        }
+        type: Object
       }
+    },
+
+    attached: function() {
+      this.sharedElements = { 'hero': this.$.header };
```

Comment:
this is different from 1.x, as at every `attached` you'd override `this.animationConfig` and `this.sharedElements`.
WDYT of just having a check before this line like `if (this.animationConfig) return;`?